deploy-agent/deployd/common/utils.py

```diff
@@ -267,7 +268,27 @@ def get_container_health_info(commit, service, redeploy) -> Optional[str]:
                                             tags={"status": "healthy", "service": service, "commit": commit})
             return returnValue
         else:
-            return None
+            fn = os.path.join("/mnt/deployd/", "{}_HEALTH_CHECK".format(service))
```

Comment:
can you create a helper fucntion for this code part?

Reply:
That sounds good. I'll proceed with updating the PR next Monday.